populate_dev_data_handler.go

```diff
@@ -68,6 +73,41 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
         },
     }
 
+    // Get the redirects, but don't follow them. Mimics http.ErrUseLastResponse (golang v1.7)
+    client := urlfetch.Client(ctx)
+    client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+        return errUseLastResponse
+    }
+
+    for _, browserName := range([]string {
+        "chrome",
+        "edge",
+        "firefox",
+        "safari",
+    }) {
+        jsonURL := "https://wpt.fyi/json?platform=" + browserName
+        resp, err := client.Head(jsonURL)
+
+		if urlError, ok := err.(*url.Error); !ok || urlError.Err != errUseLastResponse {
+			fmt.Fprintf(w, "Failed to fetch latest run for %s: %s\n", browserName, urlError.Error())
+			continue
+		}
```

Comment:
Looks like the indentation is inconsistent 4 vs. 8

Reply:
Also this `if` statement seems unnecessarily indented?

Reply:
Not sure where the bad intending is coming from - Done.